.github/workflows/ci.yml

```
@@ -84,39 +93,15 @@
         with:
           token: ${{ steps.app-token-push.outputs.token || secrets.GH_TOKEN_PUSH }}
-          fetch-depth: 0 # Required if you’ll push to an existing branch
+          fetch-depth: 0
 
-      # Only run version increase on push to main, not on pull requests
       - name: Increase Version
-        env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_PUSH }}
         id: version_step
-        if: github.event_name == 'push' && github.ref == 'refs/heads/v2'
         uses: ./
```

actionlint (1.7.7)

missing input "github_token" which is required by action "Version Update" defined at "./". all required inputs are "github_token"

(action)